macros/WebExtAllExamples.ejs

```diff
@@ -0,0 +1,60 @@
+
+<%
+
+/*
+The "examples.json" file from the https://github.com/mdn/webextensions-examples
+repository contains information about the examples living in that repository.
+
+This macro loads the "examples.json" file and uses it to build a table
+presenting that information.
+*/
+
+const examplesBaseUrl = "https://github.com/mdn/webextensions-examples/tree/master/";
+const allExamples = mdn.fetchJSONResource("https://raw.githubusercontent.com/mdn/webextensions-examples/master/examples.json");
+const lang = env.locale;
+
+const s_name = mdn.localString({
+    "en-US": "Name"
+});
+
+const s_desc = mdn.localString({
+    "en-US": "Description"
+});
+
+const s_javascript_apis = mdn.localString({
+    "en-US": "JavaScript APIs"
+});
+
+function writeJavaScriptAPIs(apisJSON) {
+  let output = "";
+  for (let jsAPI of apisJSON) {
+    var link = `/${lang}/Add-ons/WebExtensions/API/${jsAPI.replace(".", "/")}`;
```

Comment:
Some of the locales (Korean, Greek) have translated the slug for Add-ons.  They are currently being standardized because of the vanity URL rewriting, but this will break if this feature is removed.  I think it is OK for today, we can let future devs figure it out.